### PR TITLE
Remove get new tasks functionality

### LINKED ISSUE #707: Delete the old staffing code
There's a get new tasks button that is hidden by default on the Orchestra dashboard. It hooks into logic that automatically assigns you a new task. The work we've done with StaffBot and auto-staffing largely obviates the need for this button and the logic behind it. We should clean up that code once we ship v0 of autostaffing.

- [x] settings.ORCHESTRA_ENABLE_NEW_TASK_BUTTONS
- [x] enable_new_task_buttons
- [x] enableNewTaskButtons
- [x] newTasks
- [x] preventNew (not just on backend but in both old and new Orchestra frontends)
- [x] noTaskTimer
- [x] newTaskError
- [x] new_task_assignment (urls.py path: /orchestra/api/interface/)
- [x] Exceptions these functions used to throw and were only used by these functions

### PR DESCRIPTION
**Pull request by marcua**
With the introduction of StaffBot, the Available Tasks pane, and auto-staffing, we've now made the "Get new tasks" button unnecessary. This is a button that used to be displayed on the Orchestra dashboard and you could click to get a new task if one was available. Both the model of work (guess when you can click the button, get assigned a random task) and the functionality itself are not in line with how we distribute work within B12, and we've been running for years with `settings.ORCHESTRA_ENABLE_NEW_TASK_BUTTONS = False` in production.

This PR removes the button as well as the implementation for "give me a random task if one is available." The PR updates both the old Angular frontend and new React frontend.

Testing
* [x] a `git grep` for each item on the list in #707 is empty.

Old/Angular
  * [x] Orchestra dashboard loads with no errors/exceptions.
  * [x] Orchestra task loads with no errors/exceptions. 
  * [x] Time-tracking works with no errors/exceptions.
  * [x] Getting a new task via Available Tasks still works with no errors/exceptions.
  * [x] Assignment of experts via project management still works with no errors/exceptions.
  * [x] Assignment of experts via the task card still works with no errors/exceptions.
  * [x] The Staff/Restaff buttons run successfully with no errors/exceptions.

New/React
  * [x] Orchestra dashboard loads with no errors/exceptions.

Resolves #707

### CODE REVIEW

File: orchestra/workflow/certifications.py [hunk 1]
```diff
@@ -40,8 +40,8 @@ def migrate_certifications(source_workflow_slug,
                 slug=certification_slug)
         except Certification.DoesNotExist:
             logger.exception(
-                ("Certification {} doesn't exist for the destination workflow"
-                    .format(certification_slug, destination_workflow_slug)))
+                ("Certification {} doesn't exist for the destination workflow",
```
Comment (marcua) [inline orchestra/workflow/certifications.py:43]: fix an unrelated-to-this-PR error my linter caught

File: example_project/example_project/orchestra_settings.py [hunk 2]
```diff
@@ -226,10 +226,6 @@ def setup_orchestra(settings_module_name):
     # tasks while existing ones are returned by reviewers.
     settings.ORCHESTRA_ENFORCE_NO_NEW_TASKS_DURING_REVIEW = True
 
-    # Set to True if you want to enable buttons to pull new
-    # delivery/review tasks in the dashboard.
-    settings.ORCHESTRA_ENABLE_NEW_TASK_BUTTONS = True
```
Comment (marcua) [inline example_project/example_project/orchestra_settings.py:231]: irrelevant as the new task button is gone

File: orchestra/api_urls.py [hunk 3]
```diff
@@ -44,10 +43,6 @@
         submit_task_assignment,
         name='submit_task_assignment'),
 
-    url(r'^interface/new_task_assignment/(?P<task_type>\w+)/$',
```
Comment (marcua) [inline orchestra/api_urls.py:47]: this is the endpoint the button called

File: orchestra/frontend/build/asset-manifest.json [hunk 4]
```diff
@@ -1,25 +1,25 @@
 {
```
Comment (marcua) [inline orchestra/frontend/build/asset-manifest.json:1]: all changes in orchestra/frontend are rebuilds of ORCHESTRA_URL/orchestra/newapp after we removed `preventNew`

File: orchestra/project_api/tests/test_project_api.py [hunk 5]
```diff
@@ -32,7 +32,7 @@
 from orchestra.tests.helpers.fixtures import WorkflowVersionFactory
```
Comment (marcua) [inline orchestra/project_api/tests/test_project_api.py:32]: The changes to tests largely revolve around the fact that we'd call the HTTP/Python APIs for `new_task_assignment` in various places to pick up a task for subsequent testing, and now that we removed those APIs, we more directly just call `assign_task`. In hindsight, this change was probably the better implementation all along, as it's a more direct way to get a task/worker in the correct configuration to run a test.

File: orchestra/static/dist/main.js [hunk 6]
```diff
@@ -47608,7 +47608,6 @@ function orchestraTasks($http) {
     data: null,
```
Comment (marcua) [inline orchestra/static/dist/main.js:47608]: all changes to javascript here are to remove the preventNew/get new task functionality in the old Angular version of the experience.

**Review by adbharadwaj — APPROVED** [review]
a!

**Pull request merged**